### DISCUSSION TITLE
reference/docs/oracle

### DISCUSSION THREAD

**Discussion by giscus** [jaywcjlove/reference#563] [Q&A]
# reference/docs/oracle

https://wangchujiang.com/reference/docs/oracle.html

**Comment by 0xAC7** [comment]
感觉 merge，开窗函数，行转列，列转行，字段逗号分隔转多行，多行转逗号分隔都挺常用的
还有一些 pivot，cube 等处理

**Reply by jaywcjlove** [reply]
@0xAC7 欢迎提交 PR
